### ISSUE TITLE
T6: /status — host-level view of running trainings (compile phase, loss trend, GPU, disk)

### ISSUE THREAD

**Issue by thodel** [thodel/agentic_historian#366]
`/train_status` in #363 answers "what is job X doing" — it reads the job record and reports
a stage. Today's shard-00 run showed that this is not enough to tell whether a training is
healthy, and that the missing facts are all host-level.

Worked example from 2026-08-10/11. A `/train_status` would have said `running / compile`
for nine hours straight. What actually mattered:

| observation | what it meant | what a stage field showed |
|---|---|---|
| RSS climbing 2 GB/min, no output file | still in `compile`'s **parse** phase, memory grows with pages | `compile` |
| RSS flat at 63.8 GB, temp arrow growing 1 GB/5 min | **extract** phase, healthy, ETA computable | `compile` |
| RSS at 129 GB with 76 GB free | about to be killed | `compile` |
| `train_loss` flat at 27,000 for 11 epochs | learning nothing, LR stuck in 1cycle warmup | `running / train` |
| GPU 1 at 0% with the process alive | startup / dataloader-starved, not training | `running / train` |

Every one of those was diagnosed by hand over SSH with `/proc`, `nvidia-smi` and `du`. A
`/status` command should surface them.

## Proposed command

`/status` — read-only, no admin gate (it changes nothing), one embed:

```
Training host — srv, 2026-08-11 12:04

kraken-medieval-shard00-v1     train    epoch 12/30   CER 0.61 ↓   ETA 16:20
  loss 18978 ↓   3.21 it/s   early-stop 0/15

kraken-medieval-shard01-v1     compile  extract 41 GB / ~102 GB   ETA 21:10
  rss 63.8 GB (flat)

GPU 0  10.2 GB  0%   (RAG — not ours)
GPU 1  13.3 GB  99%
disk   536 GB free (70%)      RAM 237/251 GB available
```

Rules the format encodes, each learned the hard way:

- **compile reports its phase**, `parse N/total` or `extract X GB`, not just "compile".
  They have opposite memory behaviour and only one of them can be extrapolated to an ETA.
- **an arrow direction on loss and CER.** A single value cannot distinguish "learning
  slowly" from "flat at the blank-collapse plateau"; the previous value is what carries
  the information.
- **GPU 0 is labelled as not ours.** The RAG server lives there and its memory must never
  read as a training problem.
- **RSS is shown for compile only when it is still growing.** Flat is the healthy state
  and does not need a number; growing is the state that precedes a kill.

## Data source

The bot cannot read `/proc` on the box, so this needs a gateway endpoint — the routes in
`atr_serving/api/train_routes.py` currently expose only job records:

- [ ] `GET /train/host` on the gateway: per-GPU utilisation and memory, disk free on the
      arrow/checkpoint volumes, RAM available
- [ ] per-job runtime detail on `GET /train/jobs/{id}`: compile phase + progress counters,
      last epoch metrics with the previous value for the trend arrow, it/s, early-stopping
      counter
- [ ] `TrainingClient` (#361) methods for both
- [ ] `/status` renders the embed; degrades to job records alone if `/train/host` 404s, so
      it works against an older gateway

Needs a companion issue in `thodel/serving-atr-inference` for the two endpoints — the data
does not exist there yet, and `/status` is unimplementable without it.

Depends on #361. Complements #363 rather than replacing it: `/train_status <id>` stays the
per-job detail view, `/status` is the "is the box healthy" view.

Part of #360.
